sys/usb/usbus/usbus_control.c

```diff
@@ -241,6 +241,10 @@ static int _recv_dev_setup(usbus_t *usbus, usb_setup_t *pkt)
             case USB_SETUP_REQ_SET_ADDRESS:
                 DEBUG("usbus_control: Setting address\n");
                 usbus->addr = (uint8_t)pkt->value;
+                if (!IS_ACTIVE(USBDEV_SET_ADDR_AFTER_STATUS)) {
```

Comment:
```suggestion
                if (USBDEV_SET_ADDR_AFTER_STATUS) {
```
Is sufficient if `USBDEV_SET_ADDR_AFTER_STATUS` is always set.